includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -29,10 +29,10 @@ public function getData() {
 			'header' => [
 				'type' => 'link-image',
 				'image' => 'wds-company-logo-fandom-powered-by-wikia-two-lines',
-				'href' => 'http://fandom.wikia.com',
+				'href' => $this->getHref( 'fandom-logo' ),
 				'title' => [
-					'type' => 'translatable-text',
-					'key' => 'global-footer-fandom-header'
+					'type' => 'text',
+					'value' => 'Fandom powered by Wikia'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)